<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-48G8LDM" ignored="0" name="Create Customer API Test" tests="3" failures="1" timestamp="2020-05-01T14:34:45 IST" time="9.673" errors="0">
  <testcase name="validateCreateCustomerAPIWithInvalidSecretKey" time="4.637" classname="com.w2a.APITestingFramework.testcases.CeateCustomerTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.w2a.APITestingFramework.testcases.CeateCustomerTest.validateCreateCustomerAPIWithInvalidSecretKey(CeateCustomerTest.java:40)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- validateCreateCustomerAPIWithInvalidSecretKey -->
  <testcase name="validateCreateCustomerAPIWithValidSecretKey" time="2.47" classname="com.w2a.APITestingFramework.testcases.CeateCustomerTest"/>
  <testcase name="validateCreateCustomerAPIWithValidSecretKey" time="2.263" classname="com.w2a.APITestingFramework.testcases.CeateCustomerTest"/>
</testsuite> <!-- Create Customer API Test -->
